betamax/exceptions.py

```diff
@@ -2,9 +2,6 @@ class BetamaxError(Exception):
     def __init__(self, message):
```

Comment:
We could instead set-up the `message` attribute in here. The exception repr is really only an attempt to be nicer to users.

Reply:
But the default repr looks almost entirely like this one (except for one extra comma).

Reply:
🤷‍♂️ works for me

Reply:
I'll add a test for that just to be sure.